source/icu.net/NativeMethods/NativeMethods.cs

```diff
@@ -387,12 +389,12 @@ private static IntPtr GetIcuLibHandle(string basename, int icuVersion)
 #if NET6_0_OR_GREATER
 				var errorMsg = IsWindows
 					? new Win32Exception(lastError).Message
-					: $"{lastError}";
+					: $"{lastError} ({exceptionErrorMessage})";
```

Comment:
it looks like this means that we are still throwing away the exception message on windows, could we include it there too?

Reply:
Would something like this work:
```
				if (!string.IsNullOrEmpty(exceptionErrorMessage))
				{
					exceptionErrorMessage = $" ({exceptionErrorMessage})";
				}
				var errorMsg = IsWindows
					? $"{new Win32Exception(lastError).Message}{exceptionErrorMessage}"
					: $"{lastError}{exceptionErrorMessage}";
```

Reply:
Good point, and thanks for the code snippet. I updated the PR with that code.